### PR TITLE
Swift3 support

### LINKED ISSUE #4: Swift 3 support

### PR DESCRIPTION
**Pull request by tbaranes**
Close #4

### CODE REVIEW

**Comment by lfarah** [comment]
Hey, I'd love to see this lib available in Swift 3 so I can implement it in my project.

Thanks 😄

**Comment by auspicious99** [comment]
@tbaranes , sorry if I'm missing something obvious (I'm more familiar with objective C and just using this in an objective C project), but I copied and pasted your autocompleteField.swift to use with Xcode 8, and Xcode suggested to change some public to open, private to fileprivate, etc. .. may I suggest you make those changes here and commit?

File: AutocompleteField/AutocompleteField.swift [hunk 1]
```diff
@@ -19,159 +19,160 @@ public enum AutocompleteType {
...
         }
-        
+
         // only return first word if in word mode
         if(self.autocompleteType == .Word)
         {
-            let words = self.text!.componentsSeparatedByString(" ")
-            let suggestionWords = text.componentsSeparatedByString(" ")
+            let words = self.text!.components(separatedBy: " ")
+            let suggestionWords = self.text!.components(separatedBy: " ")
```
Comment (auspicious99) [inline AutocompleteField/AutocompleteField.swift:166]: This line should not have been changed from the original:
let suggestionWords = text.componentsSeparatedByString(" ")

**Review by auspicious99 — COMMENTED** [review]
There appears to be a typo in line 166 (see my comment below), otherwise it looks good

File: AutocompleteField/AutocompleteField.swift (hunk 1 above)
Comment (auspicious99) [reply AutocompleteField/AutocompleteField.swift:166]: Oops, ok, it should be:
`let suggestionWords = text.components(separatedBy: " ")`
what I meant was, should be text, not self.text!

**Comment by tbaranes** [comment]
Since `AutocompleteField` doesn't seem to be maintained anymore, I'm closing this.

**Pull request closed without merging**